src/main/java/com/telesign/phoneid/PhoneId.java

```diff
@@ -26,19 +31,65 @@
 
 	private final String customer_id;
 	private final String secret_key;
-	private int connectTimeout = 30000;
-	private int readTimeout = 30000;
-	private String httpsProtocol = "TLSv1.2";
+	private int connectTimeout;
+	private int readTimeout;
+	private String httpsProtocol;
+	private boolean runTests;
 	
-	private static final String API_BASE_URL = "https://rest.telesign.com";
+	private String API_BASE_URL = "https://rest.telesign.com";
+	private static final String BASE_URI_REST_ND_TELESIGN = "https://rest-nd.telesign.com";

```

Comment:
I just confirmed that the documentation for ND had the wrong domain. This will be under the rest.telesign.com domain. Please update.